app/assets/stylesheets/modules/flash.scss

```diff
@@ -0,0 +1,18 @@
+@import "variables";
+
+.flash-alert {
+  background-color: rgba( $red, .7);
+}
+
+.flash-notice {
+  background-color: rgba( $black, .3);
+}
+
+.flash {
+  text-align: center;
```

Comment:
Properties should be ordered border-radius, margin-left, margin-right, max-width, padding, text-align